documentation/projects/proposals/document_all_media_properties/index.md

```diff
@@ -1,5 +1,8 @@
 # Document all media properties
 
-```{include} /_toc_templates/flat.md
+```{toctree}
+:titlesonly:
+:glob:
 
+*
```

Comment:
I think the implementation plan document is not correctly formatted:

```
/home/sara/projects/openverse/documentation/projects/proposals/document_all_media_properties/20230307-project_proposal.md:4: WARNING: Document headings start at H2, not H1 [myst.header]
/home/sara/projects/openverse/documentation/projects/proposals/document_all_media_properties/20230307-project_proposal.md:8: WARNING: Document headings start at H2, not H1 [myst.header]
/home/sara/projects/openverse/documentation/projects/proposals/document_all_media_properties/20230307-project_proposal.md:13: WARNING: Document headings start at H2, not H1 [myst.header]

```

Notice that the error is in the project proposal. The `#` at the start of that document is missing: 

https://github.com/WordPress/openverse/blob/965b9fab5762078777dd5f58621e0d7a3d99e519/documentation/projects/proposals/document_all_media_properties/20230307-project_proposal.md#L1

If the title is fixed there, that resolves one problem.

There is however another very sneaky issue! The implementation plan markdown file's full name is `"20230426-implementation_plan_documenting_media_properties.md "`, with a space at the end :open_mouth:. That appears to be causing some other issues like GitHub (and even my local editor) not treating the file as Markdown.

Another small issue is that the titles for the documents in this directory do not include a date stamp at the front like other documents. E.g.:

https://github.com/WordPress/openverse/blob/697f62f01a32cb7fcf2f4a7627650a113cba40da/documentation/projects/proposals/frontend_ui_state_cookie/20220218-project_proposal.md#L1

The last thing I noticed was that the sample media properties document's title gets rendered directly into the toctree as "Media Properties" without much context that it's meant as a sample document. Perhaps moving that into its own directory with its own index file so that it has an additional heading of "Sample documentation" or something, would be helpful.

Reply:
Thank you for the thorough investigation, @sarayourfriend ! I will try to fix the rendering